Python/pywarpx/_libwarpx.py

```diff
@@ -20,6 +20,10 @@
     # --- If mpi4py is going to be used, this needs to be imported
     # --- before libwarpx is loaded (though don't know why)
     from mpi4py import MPI
+    if MPI.sizeof(MPI.Comm) == ctypes.sizeof(ctypes.c_int):
+        MPI_Comm = ctypes.c_int
```

Comment:
Can the name be changed from `MPI_Comm` to `_MPI_Comm_type` for clarity? Also, it will need to be defined if mpi4py cannot be imported (in the `except` clause).

Reply:
Yep! I will get it fixed and pushed by tomorrow.